scripts/data_prep/convert_text_to_mds.py

```diff
@@ -226,7 +221,9 @@ def download_and_convert(
         downloading_iter = DownloadingIterable(object_names=file_names,
                                                output_folder=tmp_dir,
                                                object_store=object_store)
-        tokenizer = AutoTokenizer.from_pretrained(tokenizer_name)
+        tokenizer = AutoTokenizer.from_pretrained(tokenizer_name,
+                                                  add_bos_token=False,
+                                                  add_eos_token=False)
         tokenizer.model_max_length = 5000000000  # Hack to prevent warnings from HuggingFace
```

Comment:
Added this because if a user specifies a particular `bos_text` or `eos_text` the tokenizer should not automatically also add bos or eos token